coops-ui/src/App.vue

```diff
@@ -226,6 +228,8 @@ export default {
     storeEntityInfo (response) {
       if (response && response.data && response.data.business) {
         this.setEntityName(response.data.business.legalName)
+        this.setEntityType(EntityTypes.Coop)
```

Comment:
So with the dependency on legalTypes, i set it to 'CP' so it had no impact on the current UI. When there is data present in prod for legalType it will be `response.data.business.legalType`. I set this like this for a couple reasons..
1) To have no impact on current state
2) So there is no pressure at which this data is implemented in prod 
3) Also i wanted to avoid logic that would assign a default legalType if null since we're going to have many different types in the future and i'm not sure 1 UI would fit for all types. As well it would logic we would have to go back and change anyhow 

I completely understand if we need to hold off on this feature until the data is present, so we can make this 100% complete. Love to hear your thoughts!

Reply:
Will this code work when we have other entity types coming from the BE? Or will this task/story need additional PR(s) to function? If the latter then let's see if we can wait to get a full feature to merge.
Of course, I'm open to ideas and flexibility :)

Reply:
Absolutely, the idea will be to have the `response.data.business.legalType` capture the type and set it to store. Then we can use the entityFilter anywhere when we want to encapsulate components specific to certain entities, controlling the UI output. It's pretty straight forward and can allow the AR to scale with a growing number of entity types. When new types are added, we simply add the corresponding type to our `entityTypes` Enum ( so it is consistent throughout the codebase ) and use it where we have type specific components or elements in the dom.  Would love to sit down and chat about it! :)

Reply:
It was agreed to implement this change with data from the API, but now this PR is dependent on https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/1370 (which is planned for implementation ASAP).